app/Instructor/text/InstructorAssignmentEditProcess.php

```diff
@@ -0,0 +1,61 @@
+
+<?php
+
+$activePage = 'text';
+$isInstructor = true;
+
+?>
+
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Assignment Edit</title>
+  <link rel="stylesheet" type="text/css" href="../../stylesheets/main.css">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+</head>
+<body>
+
+<?php
+include '../../header.php';
+include '../../mobile-nav.php';
+?>
+
+<div class="main-page">
+<?php
+include "../../navigation.php";
+?>
+
+
+  <div class="main-content">
+
+
+<?php
+$getAssName = $_POST["assName"];
+$getInstructions = $_POST["Instructions"];
+$getDueDate = $_POST["dueDate"];
+$getAssId = $_POST["assId"];
+
+
+$sql="UPDATE text_assignment SET assName='".$getAssName."',Instructions='".$getInstructions."',dueDate='".$getDueDate."' where assId='". $getAssId ."'";
+//VALUES ('".$getStudId."','".$getAssId."','".$getGrade."','".$getSubId."')";
+
+if($conn->query($sql) === TRUE)
+{
+    echo "Your post was submitted. Go <a href='index.php'>back</a> or go <a href='classHome.php'>home</a>.";
```

Comment:
this `href` doesn't seem to go anywhere... it likely broke when we updated the UI.
@vjrene can you advise where they should redirect to?

(This happens several times)